net/freeradius/src/opnsense/mvc/app/controllers/OPNsense/Freeradius/forms/dialogEditFreeRADIUSUser.xml

```diff
@@ -0,0 +1,44 @@
+<form>
+    <field>
+        <id>user.enabled</id>
+        <label>Enabled</label>
+        <type>checkbox</type>
+        <help>This will enable or disable the user account. Allowed characters are 0-9, a-z, A-Z, and ._-</help>
+    </field>
+    <field>
+        <id>user.username</id>
+        <label>Username</label>
+        <type>text</type>
+        <help>Set the unique username for the user. Allowed characters are 0-9, a-z, A-Z, and ,._-!$%/()+#= with up to 128 signs</help>
```

Comment:
"signs" does only work in German, needs to be "allowed characters." (dot at the end if the first message is a sentence as well)